.github/workflows/e2e.yml

```diff
@@ -91,17 +116,26 @@ jobs:
           openssl req -new -key glauth.key -out glauth.csr -config example/tls/csr.conf
           openssl x509 -req -in glauth.csr -signkey glauth.key -out glauth.crt -extensions req_ext -extfile example/tls/csr.conf -days 365
 
-      - name: build
-        run: make build
-
       - name: run glauth
         run: |
           ./glauth -c sample.cfg &
           sleep 5
 
-      - name: test-tls
+      - name: build
+        run: |
+          make build
+          sudo mkdir -p /etc/envoy
+          sudo cp libgolang.so /etc/envoy/libgolang.so
+
+      - name: envoy
+        run: |
+          curl -L -o envoy https://github.com/envoyproxy/envoy/releases/download/v1.26.2/envoy-contrib-x86_64
```

Comment:
better use a ENV variable(or some kind of thing) instead of the hard code `v1.26.2`?

Reply:
OK